business/src/main/java/in/koreatech/business/feature_changepassword/changepassword/ChangePasswordScreen.kt

```diff
@@ -101,10 +101,10 @@ fun ChangePasswordScreen(
                         }
                         innerTextField()
                     }
-                    Divider(
-                        color = if (password.isEmpty()) Blue1 else Color.Black,
+                    HorizontalDivider(
+                        modifier = modifier.fillMaxWidth(),
```

Comment:
```suggestion
                        modifier = Modifier.fillMaxWidth(),
```
Modifier.fillMaxWidth()
로 변경해야됩니다.